src/pages/iou/IOUCurrencySelection.js

```diff
@@ -127,7 +128,10 @@ function IOUCurrencySelection(props) {
     }, [currencyList, searchValue, selectedCurrencyCode, translate]);
 
     return (
-        <ScreenWrapper includeSafeAreaPaddingBottom={false}>
+        <ScreenWrapper
+            includeSafeAreaPaddingBottom={false}
+            onEntryTransitionEnd={() => optionsSelectorRef.current && optionsSelectorRef.current.focus()}
```

Comment:
Note that this doesn't work for the case of when a screen is popped to reveal the selector. See [this PR](https://github.com/Expensify/App/pull/26415/files#diff-58e9afec1ac387c92300582afe20b46d7ae577a515cf97f5c4ba58948ad4fd46R270-R282) for an example of a reliable way to auto-focus when this screen opens after the transition finishes:

```js
const focusTimeoutRef = useRef(null);
useFocusEffect(
    useCallback(() => {
        focusTimeoutRef.current = setTimeout(() => optionsSelectorRef.current?.focus(), CONST.ANIMATED_TRANSITION);
        return () => {
            if (!focusTimeoutRef.current) {
                return;
            }
            clearTimeout(focusTimeoutRef.current);
        };
    }, []);
);
```